medusa/server/web/home/handler.py

```diff
@@ -248,23 +248,13 @@ def show_statistics():
         return show_stat, max_download_count
 
     def is_alive(self, *args, **kwargs):
-        if 'callback' in kwargs and '_' in kwargs:
-            callback, _ = kwargs['callback'], kwargs['_']
-        else:
-            return 'Error: Unsupported Request. Send jsonp request with \'callback\' variable in the query string.'
-
-        # self.set_header('Cache-Control', 'max-age=0,no-cache,no-store')
         self.set_header('Content-Type', 'text/javascript')
```

Comment:
The content type is `application/json; charset=UTF-8` now

Reply:
Is there any way we can set the charset by default?

Reply:
Don't think so. But API v2 (+ v1) set it up globally when the `Content-Type` header is set.